gammapy/data/data_store.py

```diff
@@ -189,14 +189,25 @@ def info(self, show=True):
         else:
             return s
 
-    def obs(self, obs_id):
+    def obs(self, obs_id, required_IRF="all"):
```

Comment:
Is it possible to expose this option just just in `.get_observations()`? Just as we do for `skip_missing`? I would argue that in `.obs()`, when a specific `obs_id` is given we should just return it...

Reply:
Done!